app/graphql/types/objects/consultation/base.rb

```diff
@@ -22,6 +22,7 @@ def visible?(context)
 				field :ministry,													Types::Objects::Ministry, nil, null: false
 				field :consultation_responses_count,			Integer,nil, null: false
 				field :page,															Types::Objects::CmPageBuilderType, null: true
+				field :enforce_private_responses,					Boolean, nil, null: false
```

Comment:
```suggestion
				field :enforce_private_response,					Boolean, nil, null: false
```

A consultation can have only one response, the field should be `enforce_private_response`, please document in API and UI ticket.